.gitignore

```diff
@@ -1,3 +1,4 @@
+/.idea
```

Comment:
You should use a global git ignore instead. Otherwise every project has to ignore your particular setup/ide files.

ex: https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
I'd never heard of that. Awesome!